bigtop-packages/src/charm/spark/layer-spark/tests/01-basic-deployment.py

```diff
@@ -22,14 +22,13 @@ class TestDeploy(unittest.TestCase):
     """
     Trivial deployment test for Apache Spark.
     """
-    def setUp(self):
+
+    def test_deploy(self):
```

Comment:
I can do this as part of the merge.  I had it partially done but got side tracked with meetings.